apps/framework-docs/src/pages/moose/olap/model-materialized-view.mdx

```diff
@@ -567,7 +579,7 @@ from pydantic import BaseModel
 from moose_lib import MaterializedView, MaterializedViewOptions, OlapConfig
 
 class Dest(BaseModel):
-    id: str
+    id: Key[str] ## Do not need to set `order_by_fields` for the materialized view with a Key type
```

Comment:
we still need `order by` for `ReplacingMergeTree` i think

Reply:
oh shit yes